lib/ignored-paths.js

```diff
@@ -142,25 +91,26 @@ function IgnoredPaths(options) {
      * @returns {array} raw ignore rules
      */
     function addIgnoreFile(ig, filepath) {
-        return ig.addIgnoreFile(filepath);
+        ig.ignoreFiles.push(filepath);
+        return ig.add(fs.readFileSync(filepath, "utf8"));
```

Comment:
Yep, it's being checked before this method is called, [when user specifies directly](https://github.com/eslint/eslint/pull/5461/files#diff-e400329878cdf2c545a487d7a0466e4eR130) and [when searching for it in cwd](https://github.com/eslint/eslint/pull/5461/files#diff-e400329878cdf2c545a487d7a0466e4eR50).